services/quests/src/actions/Quest.tsx

```diff
@@ -25,6 +25,21 @@ import {setSnackbar} from './Snackbar';
 const ReactGA = require('react-ga') as any;
 const QueryString = require('query-string');
 
+// Override realtime api error handling, which by default
+// calls alert() and sets window.href to "/" whenever an error occurs.
+if (realtimeUtils) {
+  realtimeUtils.__proto__.onError = (error: any) => {
+    if (error.type === window.gapi.drive.realtime.ErrorType
+        .TOKEN_REFRESH_REQUIRED) {
+      realtimeUtils.authorizer.authorize(() => {
+        console.log('Error, auth refreshed');
+      }, false);
+    } else {
+      console.error(error);
```

Comment:
I feel like we should also be logging this on our end / to Google Analytics....

Reply:
This is probably going to happen on every new quest until realtime API is perma-deprecated... let's hold off on logging this to GA for now.